packages/containers/psammead-timestamp-container/src/index.jsx

```diff
@@ -1,8 +1,8 @@
 import React from 'react';
 import { number, string, bool, shape } from 'prop-types';
 import { scriptPropType } from '@bbc/gel-foundations/prop-types';
+import moment from 'moment';
```

Comment:
Why the change from `moment-timezone`?

Reply:
https://github.com/bbc/psammead/pull/2369#discussion_r335349318